gnomad_qc/v3/create_release/prepare_vcf_data_release.py

```diff
@@ -708,6 +714,18 @@ def prepare_vcf_header_dict(
         subset_list=subset_list,
         subset_pops=pops,
     )
+
+    # update VCF header for VRS fields
+    VRS_Allele_IDs_description = "The computed identifiers for the GA4GH VRS Alleles corresponding to the values in the alleles column"
+    VRS_Starts_description = "Interresidue coordinates used as the location starts for the GA4GH VRS Alleles corresponding to the values in the alleles column"
+    VRS_Ends_description = "Interresidue coordinates used as the location ends for the GA4GH VRS Alleles corresponding to the values in the alleles column"
+    VRS_States_description = "The literal sequence states used for the GA4GH VRS Alleles corresponding to the values in the alleles column"
+
+    vcf_info_dict.update({"VRS_Allele_IDs": {"Number": "1", "Description": f"{VRS_Allele_IDs_description}"},
+                          "VRS_Starts": {"Number": "1", "Description": f"{VRS_Starts_description}"},
+                          "VRS_Ends": {"Number": "1", "Description": f"{VRS_Ends_description}"},
+                          "VRS_States": {"Number": "1", "Description": f"{VRS_States_description}"}})
+
```

Comment:
Do we usually also include "Type" in our VCFS? Also, it seems like the Number should not always be 1, as these are coming from arrays

Reply:
"Type" is automatic, but when I don't specify "Number", it will be "Number='.', I put it "Number=1" as the GA4GH assigned in their script.

Reply:
when it exports these fields what does type automatically get set to?

Reply:
Like this: 
`##INFO=<ID=VRS_Allele_IDs,Number=1,Type=String,Description="The computed identifiers for the GA4GH VRS Alleles corresponding to the values in the alleles column">
##INFO=<ID=VRS_Starts,Number=1,Type=Integer,Description="Interresidue coordinates used as the location starts for the GA4GH VRS Alleles corresponding to the values in the alleles column">
##INFO=<ID=VRS_Ends,Number=1,Type=Integer,Description="Interresidue coordinates used as the location ends for the GA4GH VRS Alleles corresponding to the values in the alleles column">
##INFO=<ID=VRS_States,Number=1,Type=String,Description="The literal sequence states used for the GA4GH…`